pm_muairss_read/get_out_folder.py

```diff
@@ -0,0 +1,27 @@
+import os
+import sys
+
+import yaml
+
+
+def filefinder(root, substr, exception='', exceptfile=''):
+    '''finds files with names containing substr
+        within folder root and its subfolders'''
+    filelst = []
+    for i in os.walk(root):
+        for j in i[2]:
+            if (substr in j and (exceptfile == '' or exceptfile not in j) and
+                    (exception == '' or exception not in i[0])):
+                filelst.append(i[0]+os.sep+j)
+    return filelst
+
+
+try:
```

Comment:
best practice is to have all scripts within the tool folder. This is because on publishing, each tool would be independent, so it would be problematic to have the script be somewhere else